config/locales/en.yml

```diff
@@ -34,7 +34,7 @@ en:
       email_sent: "Ownership transfer email sent to %{user}."
       invite_accepted: "You are the new owner of %{cookbook}."
       invite_declined: "You have declined ownership of %{cookbook}."
-    deprecated: "%{cookbook} deprecated in favor of %{replacement_cookbook}."
+    deprecated: "%{cookbook} deprecated."
```

Comment:
Removed the replacement from the internationalized success message for the flash. If there is a replacement, it already appears in the deprecation banner in the view.